apisix/http/router/radixtree_sni.lua

```diff
@@ -62,6 +62,20 @@ local function parse_pem_priv_key(sni, pkey)
 end
 
 
+local function decrypt_priv_pkey(iv, key)
+    if str_find(key, "---") then
```

Comment:
should we use `core.string.has_prefix` here?

Reply:
OK, actually i just copy and paste the original codes.